spec/money_spec.rb

```diff
@@ -157,8 +158,14 @@
     expect(Money.new(1.00).to_xml).to eq("1.00")
   end
 
-  it "returns cents in as_xml" do
-    expect(Money.new(1.00).as_xml).to eq("1.00")
+  it "returns xml in to_xml with options present" do
+    options = {  
+      :root=>"price",
+      :format=>"xml",
+      :builder=>Builder::XmlMarkup.new,
+      :skip_instruct=>true
```

Comment:
Can you use the new Ruby hash syntax throughout this PR? IE: `skip_instruct: true`.